drivers/overlay/overlay.go

```diff
@@ -1875,13 +1877,25 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	return mergedDir, nil
 }
 
+// getMergedDir returns the directory path that should be used as the mount point for the overlayfs.
+func (d *Driver) getMergedDir(id, dir string, inAdditionalStore bool) string {
+	// If the layer is in an additional store, the lock we might hold only a reading lock.  To prevent
+	// races with other processes, use a private directory under the main store rundir, where we hold
+	// a read/write lock.
```

Comment:
I don’t think we have locks “scoped to `rundir`”. Which lock is this?

Reply:
we don't have a lock for that but we are running in a critical section with a lock for writing to the storage, and the rundir cannot be shared around so I think it is safe to assume we are the only writers to it.

Reply:
I’m sorry, what I meant is: please include the exact details of this (e.g. what is the lock that is protecting this use) in the comment, so that we can find this relationship later, or notice if something ever unintentionally changed and the design broke.